Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -250,50 +250,58 @@ Set-Alias -Scope Global Rebuild-Solution Invoke-RebuildSolution
 Function Invoke-RebuildSolution {
     Param(
         [string] $buildConfiguration = "Debug",
-        [string] $target = "build",
         [string] $verbosity = "minimal",
-        [string] $maxCpuCount = "",
-        [string] $noReuse = "false",
```

Comment:
This one would probably also be nice to keep. It cleans up any msbuild worker nodes used during the build. The parameter name is confusing though. The parameter for msbuild is `nodeReuse` not `noReuse` so if we're okay with changing the parameters for this function we should probably fix the name.

Reply:
Updated